src/app/app.tsx

```diff
@@ -5,41 +5,71 @@ import { Link, Router, View } from 'react-navi';
 import { compose, mount, route, withView } from 'navi';
 import { Articles } from './components/articles/articles.component';
 import { Reports } from './components/reports/reports.component';
+import { HeaderComponent } from './components/header/header.component';
```

Comment:
rename to MainHeader; Component suffix is redundant here

Reply:
renamed